packages/client/README.md

```diff
@@ -59,3 +59,41 @@ const reactionRequest = {
 // lensClient is already authenticated from the previous step
 await lensClient.reactions.add(reactionRequest);
 ```
+
+### Set a dispatcher with gasless, signed data type
+
+```ts
+const typedDataResult = await lensClient.profile.createSetDispatcherTypedData({
+  profileId: '0x0185', // your profileId
+});
+
+// typedDataResult is a PromiseResult object
+const setDispatcherData = typedDataResult.unwrap().result;
```

Comment:
all returned from `execute` helper is a `PromiseResult` so requires checks for isSuccess, isFailure, unwrap or just `.value` (if success)

Reply:
yeah, I know, I thought the same in the beginning, then we decided with @cesarenaldi to first separate user-flow-errors (the ones we know that can happen -> result.isFailure()) from unexpected errors (they will still throw) and introduce our users to the concept of `Result` type which is common in other programming languages.

I'm working on the next PR when I'm using this Result type a lot (from an end-user perspective) so I will share my experience later.